va/va.h

```diff
@@ -1165,6 +1171,26 @@ typedef union _VAConfigAttribValContextPriority{
     uint32_t value;
 }VAConfigAttribValContextPriority;
 
+/** \brief Attribute value for VAConfigAttribLookAhead*/
+typedef union _VAConfigAttribValEncLookAhead {
+    struct {
+        /** \brief num of frames to look ahead. 0 means lookahead disabled */
+        uint32_t look_ahead_depth               : 8;
+        /** \brief max num of frames to look ahead before actual encoding pass,
+         *  application can set look_ahead_depth as (0, max_num_look_ahead_frames] */
+        uint32_t max_num_look_ahead_frames      : 8;
+        /** \brief indicate whether the current encoding is in the Look Ahead pass,
+         *  if look_ahead_phase == 0, the current encoding is in the actual encoding pass
+         *  if look_ahead_phase == 1, the current encoding is in the look ahead pass. */
+        uint32_t look_ahead_phase               : 1;
+        /** \brief A flag indicates whether look ahead is supported */
+        uint32_t look_ahead_support             : 1;
```

Comment:
look_ahead_support is enough? as I know, there are different output types. so maybe need several bits to indicate it?